content/applications/finance/payment_providers/mercado_pago.rst

```diff
@@ -5,6 +5,9 @@ Mercado Pago
 `Mercado Pago <https://www.mercadopago.com/>`_ is an online payment provider covering several
 countries, currencies and payment methods in Latin America.
 
+.. seealso::
+   `Mercado Pago Odoo webinar <https://www.youtube.com/watch?v=CX8vPHMb1ic>`_
```

Comment:
I would move this at the end of the page and merge it with the other seealso (and have 2 bullet points in there).